docs/taskruns.md

```diff
@@ -771,6 +773,29 @@ will execute with the [`default` service account](https://kubernetes.io/docs/tas
 set for the target [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
 
 For more information, see [`ServiceAccount`](auth.md).
+## `TaskRun` status
+The `status` field defines the observed state of `TaskRun`
+### The `status` field
+- Required:
+  - `status` - The most relevant information about the TaskRun's state. This field includes:
+    - `status.conditions`, which contains the latest observations of the `TaskRun`'s state. [See here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) for information on typical status properties. 
+  - `podName` - Name of the pod containing the containers responsible for executing this `task`'s `step`s.
+  - `startTime` - The time at which the `TaskRun` began executing, conforms to [RFC3339](https://tools.ietf.org/html/rfc3339) format.
+  - `completionTime` - The time at which the `TaskRun` finished executing, conforms to [RFC3339](https://tools.ietf.org/html/rfc3339) format.
+  - [`taskSpec`](tasks.md#configuring-a-task) - `TaskSpec` defines the desired state of the `Task` executed via the `TaskRun`.
+
+- Optional:
+  - `taskResults`- Contains the results of this `task`.
```

Comment:
I think this should be just `results`? 
https://github.com/tektoncd/pipeline/blob/0a4357adbeee412428102b25e7876b38a123946b/pkg/apis/pipeline/v1/taskrun_types.go#L228

Reply:
Ohh, I will update it again. Thanks!

Reply:
Thanks! lmk when it's updated and we can merge this

Reply:
Update done! (the commit message isn't very accurate) but I didn't see the point of changing it for just a typo.

Reply:
> Update done! (the commit message isn't very accurate) but I didn't see the point of changing it for just a typo.

Thanks! Would you mind squashing the 3 commits? This should be the one last thing to merge it. Thanks!

Reply:
Done!

Reply:
Thank you!